CHANGELOG.md

```diff
@@ -45,6 +45,9 @@
   - fixed app category view more and collapse button function [#805](https://github.com/eclipse-tractusx/portal-frontend/pull/805)
 - **Connector Management**:
   - fixed blank page issue on clicking on details [#925](https://github.com/eclipse-tractusx/portal-frontend/pull/925)
+- **User Management**:
+  - Fix disable button state when deselect all checkboxes [#923](https://github.com/eclipse-tractusx/portal-frontend/pull/923)
```

Comment:
Please add your change to "Unreleased" not under the already released 2.1.0-RC1, thank you:

## Unreleased

- **User Management**:
  - Fix disable button state when deselect all checkboxes [#923](https://github.com/eclipse-tractusx/portal-frontend/pull/923)

## 2.1.0-RC1

...

Reply:
I have updated changelog for bugfix for (unreleased) RC2